src/ring_theory/witt_vector/is_alg_closed.lean

```diff
@@ -0,0 +1,584 @@
+/-
+Copyright (c) 2022 Robert Y. Lewis. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Robert Y. Lewis, Heather Macbeth
+-/
+
+import field_theory.is_alg_closed.basic
+import ring_theory.witt_vector.domain
+import ring_theory.witt_vector.truncated
+import data.mv_polynomial.supported
+
+
+/-!
+# "Eigenvectors" of the Frobenius map
```

Comment:
I understand why you called this "eigenvectors" but it still seems like not the right terminology to me ... I'm not sure what to call it either, apart from
```suggestion
# Construction of a solution to some random equation involving the Frobenius map
```
(not intended as a serious suggestion ...)

Reply:
I changed the header to `# Solving equations about the Frobenius map on the field of fractions of `𝕎 k``, which really rolls off the tongue. Better? I also renamed the file to `frobenius_fraction_field.lean`, not because I really like that, but for lack of a better idea.

Reply:
Sounds good!  Let's see if the algebraists can give us a name which upgrades from "accurate" to "inspired" :)